hack/make-rules/test-integration.sh

```diff
@@ -41,27 +41,33 @@ KUBE_TEST_ARGS=${KUBE_TEST_ARGS:-}
 kube::test::find_integration_test_dirs() {
   (
     cd ${KUBE_ROOT}
-    find test/integration/${1-} -name '*_test.go' -print0 \
+    find test/integration/ -name '*_test.go' -print0 \
```

Comment:
The `-print0` behavior is lost by the time this pipe hits the `sed` so it's functionally useless.